packages/noya-state/src/reducers/symbolsReducer.ts

```diff
@@ -0,0 +1,175 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import produce from 'immer';
+import { getSymbols } from '../selectors/themeSelectors';
+import { getCurrentTab } from '../selectors/workspaceSelectors';
+import { ApplicationState } from './applicationReducer';
+
+export type SymbolsAction =
+  | [type: 'setAdjustContentOnResize', value: boolean]
+  | [type: 'setHasBackgroundColor', value: boolean]
+  | [type: 'setBackgroundColor', value: Sketch.Color]
+  | [type: 'setIncludeBackgroundColorInExport', value: boolean]
+  | [type: 'setIncludeBackgroundColorInInstance', value: boolean]
+  | [type: 'setGroupLayout', value: Sketch.InferredLayoutAxis | undefined]
+  | [type: 'setLayoutAnchor', value: Sketch.InferredLayoutAnchor]
+  | [type: 'setLayoutAnchor', value: Sketch.InferredLayoutAnchor]
+  | [type: 'setMinWidth', value: number]
+  | [type: 'setAllowsOverride', value: boolean];
+export function symbolsReducer(
+  state: ApplicationState,
+  action: SymbolsAction,
+): ApplicationState {
+  switch (action[0]) {
+    case 'setAdjustContentOnResize': {
+      const [, value] = action;
+      const filter =
```

Comment:
I think we can use `getSelectedSymbols` throughout this file to handle the filtering and current tab check